MAINTAINERS

```diff
@@ -1,8 +1,58 @@
-Solomon Hykes <solomon@docker.com> (@shykes)
-Olivier Gambier <olivier@docker.com> (@dmp42)
-Stephen Day <stephen.day@docker.com> (@stevvooe)
-Derek McGowan <derek@mcgstyle.net> (@dmcgowan)
-Richard Scothern <richard.scothern@gmail.com> (@richardscothern)
-Aaron Lehmann <aaron.lehmann@docker.com> (@aaronlehmann)
+# Distribution maintainers file
+#
+# This file describes who runs the docker/distribution project and how.
+# This is a living document - if you see something out of date or missing, speak up!
+#
+# It is structured to be consumable by both humans and programs.
+# To extract its contents programmatically, use any TOML-compliant parser.
+#
+# This file is compiled into the MAINTAINERS file in docker/opensource.
+#
+[Org]
+	[Org."Core maintainers"]
```

Comment:
Cool, no problem, and happy to add some information; we want to standardize formats and use the docker/opensource repository for keeping the global "rules" / "process" information for all docker open source projects. This prevents duplication, but also having the "parseable" format for maintainers can be used to automate some tasks where needed (e.g. checking LGTMs on a PR, but only if they were given by a maintainer). We're working on some other uses, if you're in the office, ask Arnaud.